Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -521,6 +518,21 @@ private PlatformInterface CreatePlatformInterface(EOSConfig configData)
                 platformSpecifics.ConfigureSystemPlatformCreateOptions(ref platformOptions);
 
                 return PlatformInterface.Create(ref (platformOptions as EOSCreateOptions).options);
+
+                PlatformInterface platformInterface = platformSpecifics.CreatePlatformInterface(platformOptions);
+
+#if !(UNITY_STANDALONE_WIN || UNITY_EDITOR_WIN || UNITY_STANDALONE_LINUX || UNITY_EDITOR_LINUX || UNITY_STANDALONE_OSX || UNITY_EDITOR_OSX)
+                integratedPlatformOptionsContainer.Release();
+#endif
+                if (platformInterface != null)
+                {
+                    return platformInterface;
+                }
+                else
+                {
+                    print($"Error creating platform interface:");
```

Comment:
I don't believe this needs to be here, because we throw an exception if the platform interface is NULL int `Init`.

Reply:
That is true.  Updated.